### ISSUE TITLE
{.compileTime.} CritBitTree[T].incl fails to add keys after the 1st one

### ISSUE THREAD

**Issue by akavel** [nim-lang/Nim#11761]
Function `CritBitTree[T].incl` (and also `.containsOrIncl`) works only once when run in a `static:` block. Any further (2nd and later) calls of `.incl` do nothing. 

### Example

```nim
import critbits

static:
  var t: CritBitTree[void]
  t.incl("foo")
  t.incl("bar")
  t.incl("baz")
  echo $t

if true:
  var t: CritBitTree[void]
  t.incl("foo")
  t.incl("bar")
  t.incl("baz")
  echo $t
```

### Current Output
```
C:\prog\dali>nim c -r mytest.nim
Hint: used config file 'C:\dnload\nim-etc\nim\config\nim.cfg' [Conf]
Hint: system [Processing]
Hint: widestrs [Processing]
Hint: io [Processing]
Hint: mytest [Processing]
Hint: critbits [Processing]
{"foo"}
CC: mytest.nim
Hint:  [Link]
Hint: operation successful (14556 lines compiled; 0.641 sec total; 15.957MiB peakmem; Debug Build) [SuccessX]
Hint: C:\prog\dali\mytest.exe  [Exec]
{"bar", "baz", "foo"}
```

### Expected Output

```
C:\prog\dali>nim c -r mytest.nim
Hint: used config file 'C:\dnload\nim-etc\nim\config\nim.cfg' [Conf]
Hint: system [Processing]
Hint: widestrs [Processing]
Hint: io [Processing]
Hint: mytest [Processing]
Hint: critbits [Processing]
{"bar", "baz", "foo"}
CC: mytest.nim
Hint:  [Link]
Hint: operation successful (14556 lines compiled; 0.641 sec total; 15.957MiB peakmem; Debug Build) [SuccessX]
Hint: C:\prog\dali\mytest.exe  [Exec]
{"bar", "baz", "foo"}
```

### Additional Information

```
C:\prog\dali>nim -v
Nim Compiler Version 0.20.0 [Windows: amd64]
Compiled at 2019-06-06
Copyright (c) 2006-2019 by Andreas Rumpf

git hash: e7471cebae2a404f3e4239f199f5a0c422484aac
active boot switches: -d:release
```

I am using CritBitTree in my https://github.com/akavel/dali project, and I want to be able to use it during compile time, for building a `classes.dex` file, for development of Android .apk files in Nim.

**Comment by akavel** [comment]
I downloaded 0.20.2 just now, seems to still be present:

```
C:\prog\dali>nim c -r mytest.nim
Hint: used config file 'C:\dnload\nim-etc\nim\config\nim.cfg' [Conf]
Hint: system [Processing]
Hint: widestrs [Processing]
Hint: io [Processing]
Hint: mytest [Processing]
Hint: critbits [Processing]
{"foo"}
CC: stdlib_assertions.nim
CC: stdlib_io.nim
CC: stdlib_system.nim
CC: stdlib_critbits.nim
CC: mytest.nim
Hint:  [Link]
Hint: operation successful (14652 lines compiled; 5.090 sec total; 16MiB peakmem; Debug Build) [SuccessX]
Hint: C:\prog\dali\mytest.exe  [Exec]
{"bar", "baz", "foo"}

C:\prog\dali>nim --version
Nim Compiler Version 0.20.2 [Windows: amd64]
Compiled at 2019-07-17
Copyright (c) 2006-2019 by Andreas Rumpf

git hash: 88a0edba4b1a3d535b54336fd589746add54e937
active boot switches: -d:release
```

**Comment by narimiran** [comment]
This has been fixed in the latest devel (v1.1.1) and it will be part of the next stable release (v.1.2.0).
